android/detekt.yml

```diff
@@ -158,7 +158,7 @@ complexity:
   active: true
   LongMethod:
     active: true
-    threshold: 20
+    threshold: 30
```

Comment:
double check, maybe you can simplify.
Otherwise you will update it every time, when needed.

Reply:
Spent a long time on this before resorting to increasing the value. A constructor with named parameters in new lines doesn't increase complexity but adds up to reach this limit very quickly. If detekt supported this number to be applied to individual instructions only, I'd not increase.